frontend/src/components/LayoutItem.module.css

```diff
@@ -0,0 +1,37 @@
+.item {
+  cursor: pointer;
+  background-color: #c5ced2;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0px 20px 20px 0px;
+  flex-grow: 1;
+  padding: 0 15px;
+  text-decoration: none;
+  color: #000;
+}
+
+.item_top {
+  border-top-left-radius: 20px;
+}
+
+.item_text {
+  font-size: 24px;
+  text-wrap: wrap;
+  text-align: center;
+  text-transform: uppercase;
+  max-width: 150px;
+}
+
+.item_active {
+  background-color: #a1b2b9;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0px 20px 20px 0px;
+  flex-grow: 1;
+  padding: 0 15px;
+  text-decoration: none;
+  color: #000;
```

Comment:
i see many of these properties are the same as in `.item` class. Please remove them from here, and add the item class where it's needed